pyproject.toml

```diff
@@ -12,6 +12,7 @@ name = "Discourse2Jira"
 version = "0.1"
 dependencies = [
     "jira==3.8.0",
+    "dsctriage @ git+https://github.com/lvoytek/discourse-triage@1.4.1",
```

Comment:
Vendored version is 1.4.1. Pinning that version for now. At some point we can try to use 1.6.1 as it brings some interesting features